vehicle-tracker/vehicle_tracker_app/lib/features/home/pages/map_page.dart

```diff
@@ -0,0 +1,56 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:get/get.dart';
+import 'package:latlong2/latlong.dart';
+
+import '../../../constants.dart';
+import '../widgets/map_marker.dart';
+
+class MapPage extends StatefulWidget {
+  const MapPage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  State<MapPage> createState() => _MapPageState();
+}
+
+class _MapPageState extends State<MapPage> {
+  LatLng? markerPosition = Get.arguments['position'];
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        appBar: AppBar(
+          title: const Text("Open Street Map Preview"),
```

Comment:
use i18 key with translators

Reply:
As per the new Figma designs. This map page is not required in the app so I am removing it.